guides/micronaut-security-session/kotlin/src/main/kotlin/example/micronaut/LoginAuthController.kt

```diff
@@ -12,13 +12,9 @@ class LoginAuthController {
 
     @Get("/auth") // <3>
     @View("auth") // <4>
-    fun auth(): Map<String, Any> {
-        return HashMap()
-    }
+    fun auth(): Map<String, Any> = emptyMap()
```

Comment:
if Kotlin `emptyMap() `is immutable we should not do that. We should not return an inmutable Map as the model. Because, we support a feature called ViewModelProcessors which allows you to decorate a model. That it is to say add stuff to the map.